client/src/pages/Result.js

```diff
@@ -0,0 +1,11 @@
+import "../pages/Result.css"
+import MilestoneTable from "../component/MilestoneTable"
+const Result = () => {
+  return (
+   <div className="Result">
+   <MilestoneTable />
+   </div> 
+  )
+}
+
+export default Result ;
```

Comment:
here same as above formatting the code will be a good option

Reply:
Great point, Fixed